### ISSUE TITLE
Node 11 build

### ISSUE THREAD

**Issue by jonyrock** [hastic/hastic-server#753]
It is a piece of log during build:

```
                                     ^
/home/alex/.node-gyp/11.15.0/include/node/v8config.h:326:3: note: in definition of macro ‘V8_DEPRECATED’
   declarator __attribute__((deprecated(message)))
   ^
In file included from ../binding.cc:40:0:
../../nan/nan.h: In constructor ‘Nan::Utf8String::Utf8String(v8::Local<v8::Value>)’:
../../nan/nan.h:1034:53: warning: ‘v8::Local<v8::String> v8::Value::ToString() const’ is deprecated: Use maybe version [-Wdeprecated-declarations]
       v8::Local<v8::String> string = from->ToString();
                                                     ^
In file included from ../binding.cc:24:0:
/home/alex/.node-gyp/11.15.0/include/node/v8.h:10251:15: note: declared here
 Local<String> Value::ToString() const {
               ^
In file included from ../binding.cc:40:0:
../../nan/nan.h:1044:74: warning: ‘int v8::String::WriteUtf8(char*, int, int*, int) const’ is deprecated: Use Isolate* version [-Wdeprecated-declarations]
         length_ = string->WriteUtf8(str_, static_cast<int>(len), 0, flags);
                                                                          ^
In file included from /home/alex/.node-gyp/11.15.0/include/node/v8.h:26:0,
                 from ../binding.cc:24:
/home/alex/.node-gyp/11.15.0/include/node/v8.h:2754:21: note: declared here
                 int WriteUtf8(char* buffer, int length = -1,
                     ^
/home/alex/.node-gyp/11.15.0/include/node/v8config.h:326:3: note: in definition of macro ‘V8_DEPRECATED’
   declarator __attribute__((deprecated(message)))
   ^
../binding.cc: In member function ‘void zmq::Socket::NotifyReadReady()’:
../binding.cc:394:73: warning: ‘v8::Local<v8::Value> Nan::MakeCallback(v8::Local<v8::Object>, v8::Local<v8::Function>, int, v8::Local<v8::Value>*)’ is deprecated [-Wdeprecated-declarations]
     Nan::MakeCallback(this->handle(), callback_v.As<Function>(), 0, NULL);
                                                                         ^
In file included from ../binding.cc:40:0:
../../nan/nan.h:929:46: note: declared here
   NAN_DEPRECATED inline v8::Local<v8::Value> MakeCallback(
                                              ^
```

**Comment by rozetko** [comment]
We use zeromq `4.6.0`
Node 10 support was introduced in this PR https://github.com/zeromq/zeromq.js/pull/262
It's released in `5.0.0`
There are commits with Node 11 support but I'm not sure it's already published

P.S. the latest zeromq release was published about a year ago (Oct 18, 2018)

**Comment by jonyrock-back** [comment]
Same thing with node v13.5.0

```
../binding.cc: In static member function ‘static Nan::NAN_METHOD_RETURN_TYPE zmq::Socket::Sendv(Nan::NAN_METHOD_ARGS_TYPE)’:
../binding.cc:1295:39: error: no matching function for call to ‘v8::Array::Get(uint32_t&)’
       Local<Object> buf = batch->Get(i).As<Object>();
                                       ^
In file included from ../binding.cc:24:0:
/home/corp/.cache/node-gyp/13.5.0/include/node/v8.h:3594:43: note: candidate: v8::MaybeLocal<v8::Value> v8::Object::Get(v8::Local<v8::Context>, v8::Local<v8::Value>)   
   V8_WARN_UNUSED_RESULT MaybeLocal<Value> Get(Local<Context> context,
                                           ^~~
/home/corp/.cache/node-gyp/13.5.0/include/node/v8.h:3594:43: note:   candidate expects 2 arguments, 1 provided
/home/corp/.cache/node-gyp/13.5.0/include/node/v8.h:3597:43: note: candidate: v8::MaybeLocal<v8::Value> v8::Object::Get(v8::Local<v8::Context>, uint32_t)
   V8_WARN_UNUSED_RESULT MaybeLocal<Value> Get(Local<Context> context,
                                           ^~~
/home/corp/.cache/node-gyp/13.5.0/include/node/v8.h:3597:43: note:   candidate expects 2 arguments, 1 provided
../binding.cc:1295:50: error: expected primary-expression before ‘>’ token
       Local<Object> buf = batch->Get(i).As<Object>();
                                                  ^
../binding.cc:1295:52: error: expected primary-expression before ‘)’ token
       Local<Object> buf = batch->Get(i).As<Object>();
                                                    ^
../binding.cc:1296:48: error: no matching function for call to ‘v8::Array::Get(uint32_t)’
       Local<Number> flagsObj = batch->Get(i + 1).As<Number>();
```
